tests/scripts/yotta-build.sh

```diff
@@ -1,13 +1,31 @@
 #!/bin/sh
 
-# Do test builds of the yotta module for all supported targets
+# yotta-build.sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2014-2016, ARM Limited, All Rights Reserved
```

Comment:
Fix the copyright dates. The date should span from when the script was first created to today.